### PR TITLE
[ioctl] Build action read command line into new ioctl

### LINKED ISSUE #3321: [ioctl] build action read command line into new ioctl
### What would you like to be added
 - Step 1: create `actionread.go` and `actionread_test.go` files in `./ioctl/newcmd/action`. create this directory if not exist. 
 - Step 2: refactor  command  https://github.com/iotexproject/iotex-core/blob/master/ioctl/cmd/action/actionread.go in `actionread.go`
 - Step 3: add unit test in xx_test.go

#### Notes:
1. Use [client interface](https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go) to construct the Cobra command.
2. [Output package](https://github.com/iotexproject/iotex-core/blob/master/ioctl/output) is deprecated. `e.g.` `output.PrintError` could be replaced with `errors.Wrap`.
3. Global variables (config.ReadConfig  config.DefaultConfigFile etc.) have been wrapped into client interface. Please use [client interface](https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go) to access them if needed
4. The logic of new code should be similar to [the old one](https://github.com/iotexproject/iotex-core/blob/master/ioctl/cmd/action/actionread.go).
5. In each unit test, the results should be checked whether they are as the expected. 

#### Example:
1. https://github.com/iotexproject/iotex-core/blob/master/ioctl/newcmd/account/accountinfo.go
2. https://github.com/iotexproject/iotex-core/blob/master/ioctl/newcmd/account/accountdelete.go
3. https://github.com/iotexproject/iotex-core/blob/master/ioctl/newcmd/account/accountcreateadd.go

### Why is this needed
Unit tests couldn't be added for the old ioctl command (./ioctl/cmd)

### PR DESCRIPTION
**Pull request by LuckyPigeon**
# Description

Refactor `actionread` command in new `ioctl`, with the following note.

* Use [client interface](https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go) to construct the Cobra command.
* Output package is deprecated, replace it with errors package.
* Replace fmt.Println with cmd.Println
* Refactor unit test to cover the modification.

Fixes #3321 

## Type of change
Please delete options that are not relevant.
- [x] Code refactor or improvement

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] make test
- [x] TestNewActionReadCmd

**Test Configuration**:
- Firmware version: Windows 10 (WSL Version: Ubuntu 18.04)
- Hardware:
- Toolchain:
- SDK:

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
****

### CODE REVIEW

**Comment by LuckyPigeon** [comment]
@huof6829 I need 4 PRs in the hacktoberfest event, this is PR 4. Could you add a label of `HACKTOBERFEST-ACCEPTED` in this PR? Thanks!

File: ioctl/newcmd/action/actionread.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+// Multi-language support
+var (
+	_readCmdShorts = map[config.Language]string{
+		config.English: "read smart contract on IoTeX blockchain",
+		config.Chinese: "读取IoTeX区块链上的智能合约",
+	}
+	_readCmdUses = map[config.Language]string{
+		config.English: "read (ALIAS|CONTRACT_ADDRESS) -b BYTE_CODE [-s SIGNER]",
+		config.Chinese: "reads (别名|联系人地址) -b 类型码 [-s 签署人]",
```
Comment (huof6829) [inline ioctl/newcmd/action/actionread.go:30]: read

File: ioctl/newcmd/action/actionread.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+// NewActionReadCmd represents the action Read command
+func NewActionReadCmd(client ioctl.Client) *cobra.Command {
+	use, _ := client.SelectTranslation(_readCmdUses)
+	short, _ := client.SelectTranslation(_readCmdShorts)
+
+	cmd := &cobra.Command{
+		Use:   use,
+		Short: short,
+		RunE: func(cmd *cobra.Command, args []string) error {
```
Comment (huof6829) [inline ioctl/newcmd/action/actionread.go:42]: add `Args` checking before `RunE`

File: ioctl/newcmd/action/actionread.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+			cmd.SilenceUsage = true
+			contract, err := alias.IOAddress(client, args[0])
+			if err != nil {
+				return errors.Wrap(err, "failed to get contract address")
+			}
+			bytecodeFlag, err := cmd.Flags().GetString(bytecodeFlagLabel)
+			if err != nil {
+				return errors.Wrap(err, "failed to get flag bytecode")
+			}
+			fmt.Println(bytecodeFlag)
```
Comment (huof6829) [inline ioctl/newcmd/action/actionread.go:52]: delete

File: ioctl/newcmd/action/actionread.go [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+			result, err := Read(client, contract, "0", bytecode, signer, gasLimit)
+			if err != nil {
+				return errors.Wrap(err, "failed to Read contract")
+			}
+			cmd.Println(result)
+			return err
+		},
+	}
+	registerBytecodeFlag(client, cmd)
+	RegisterWriteCommand(client, cmd)
```
Comment (huof6829) [inline ioctl/newcmd/action/actionread.go:70]: Just required `-b` and `-s`  (line#29), not all flags

**Review by huof6829 — APPROVED** [review]

**Review by huangzhiran — APPROVED** [review]

**Pull request merged**